### ISSUE TITLE
[BUG] Error after updating to 1.5.6 and HUD does not open

### ISSUE THREAD

**Issue by nschoenwald** [Larkinabout/fvtt-token-action-hud-dnd5e#97]
**Describe the bug**
<img width="433" alt="image" src="https://github.com/Larkinabout/fvtt-token-action-hud-dnd5e/assets/72390280/05aaec38-bc3c-4605-bbc5-47f1e7a804b2">

**Versions:**
- Token Action HUD Core: 1.5.5
- Token Action HUD System: dnd5e 1.5.6
- System: dnd5e 3.1
- Foundry VTT: 11.315
- Browser: Chrome latest

**Comment by Larkinabout** [comment]
Can you click on that top most link and screenshot where it's underlined in red please.

**Comment by nschoenwald** [comment]
Huh. I had rolled the update back, then updated again just now to make the screenshot. I don't have the error this time after updating for a second time. It seems to work just fine now from a quick glance.

**Comment by Larkinabout** [comment]
I'll keep the issue open for now as it might be for a specific token. I suspect it's something to do with the label used for the tooltip on uses, so it might only be erroring for a specific item.

**Comment by morepurplemorebetter** [comment]
The only change that includes a new call to `label` that I can think of that could cause this is [line 1364 in action-handler.js](https://github.com/Larkinabout/fvtt-token-action-hud-dnd5e/blob/main/scripts/action-handler.js#L1364). Adding a `?` would probably resolve this issue and can't hurt.

Or better yet, change the line to:
```js
const period = CONFIG.DND5E.limitedUsePeriods[uses.per]?.label ?? uses.per
```

**Comment by nschoenwald** [comment]
Other people on reddit had this issue as well: https://www.reddit.com/r/FoundryVTT/comments/1dbs012/dnd5e_token_action_hud_not_working_for_2_player/

**Comment by Larkinabout** [comment]
Should be resolved in 1.5.7. Thanks.
